src/component/base/DraftEditor.react.js

```diff
@@ -190,6 +191,10 @@ class DraftEditor extends React.Component {
       if (!this.props.readOnly) {
         const method = this._handler && this._handler[eventName];
         method && method(this, e);
+      } else if (eventName === 'onCopy') {
+        editOnSelect(this)
```

Comment:
Ah, I see why. When readOnly is set the control is no longer content editable and React does not fire `onSelect` except on form controls.

We should add a comment explaining this.

Alternatively, rather than edit Draft can we set up our own copy event listener in the editor? Making changes to Draft increases the work we'll have to do when a new version is pushed (as is, we're on 10.0 and still haven't migrated to 10.1).